File: old/src/client/sripts/paths.js
class Paths {
    static MAIN = 'http://127.0.0.1:5500/';
    static BACKUPS = Paths.MAIN + 'backups/';
    static QUERIES = Paths.MAIN + 'queries/';
    static TEMP = Paths.MAIN + 'temp/';
    static CONFIG = Paths.MAIN + 'config/';
    static SOURCE = Paths.MAIN + 'src/';
    static CLIENT = Paths.SOURCE + 'client/';
    static SERVER = Paths.SOURCE + 'server/';
    static ASSETS = Paths.CLIENT + 'assets/';
    static SCRIPTS = Paths.CLIENT + 'scripts/';
    static STYLES = Paths.CLIENT + 'styles/';
    static TEMPLATES = Paths.CLIENT + 'templates/';
}

export { Paths };

File: new/src/server/paths.py

__all__ = ['Paths']
__author__ = 'Felipe Amaral'
__version__ = '0.2'

from pathlib import Path


class Paths:
    """
    Class that defines paths to various directories in the project.
    Uses direct Path objects for better clarity and maintainability.
    """


    MAIN = Path(__file__).parents[2]
    TEMP = MAIN / 'temp'
    SOURCE = MAIN / 'src'
    CLIENT = SOURCE / 'client'
    SERVER = SOURCE / 'server'
    ASSETS = CLIENT / 'assets'
    SCRIPTS = CLIENT / 'scripts'
    STYLES = CLIENT / 'styles'
    TEMPLATES = CLIENT / 'templates'
    